blazor/datagrid/template-excel-export.md

```diff
@@ -0,0 +1,590 @@
+---
+layout: post
+title: Excel exporting with Templates in Blazor Grid Component | Syncfusion
+description: Checkout and learn here all about Template Excel Export in Syncfusion Blazor DataGrid component and much more.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Excel Exporting with Templates
+
+## Export template columns
+
+The excel export provides an option to export template columns of the DataGrid by defining the [IncludeTemplateColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html#Syncfusion_Blazor_Grids_ExcelExportProperties_IncludeTemplateColumn) of [ExcelExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html) as true. In the following sample, the CustomerID column is a template column. The template values cannot be directly exported into the cells. To customize the values of the template columns in Excel file, you must use [ExcelQueryCellInfoEvent](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_ExcelQueryCellInfoEvent).
```

Comment:
first line: The excel export provides an option to export template columns of the DataGrid by defining the [IncludeTemplateColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html#Syncfusion_Blazor_Grids_ExcelExportProperties_IncludeTemplateColumn) of [ExcelExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html) as true.

Second paragraph: The excel export functionality allows you to export Grid columns that include images, hyperlinks, and custom text to an Excel document.

Third Paragraph: In the following sample, the CustomerID column is a template column. The template values cannot be directly exported into the cells.To customize the values of the template columns in Excel file, you must use [ExcelQueryCellInfoEvent]

Reply:
Use above paragraph order. Don't say all in single paragraph. 

Also, use the proper casing in some places excel word used as small case, third paragraph Excel used as upper case.

Reply:
Proper casing is `Excel` don't use `excel`